config/credentials.default.coffee

```diff
@@ -1,3 +1,5 @@
+secret = () -> Math.random().toString(36).slice(-16)
```

Comment:
Since credentials.default.coffee would be editted by deployers, i dont think it is necessary and we dont need to worry about. With the same logic, we would need to generate each credential (postgres, mongo username/pass etc) for each deployment. Our default credential config is not safe by default like every other default config.